### PR TITLE
complete question 6 no error handling

### PR DESCRIPTION
**Pull request by ID1203**
question 6 but no error handling waiting on pull request from before

### CODE REVIEW

File: __tests__/intergration.test.js [hunk 1]
```diff
@@ -66,5 +66,38 @@ describe('/api/articles/:article_id', () => {
...
-
+describe('/api/articles/:article_id/comments', () => {
+    it('should return comments of article based on id', () => {
+        return request(app).get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+            console.log(response.body);
+            expect(Array.isArray(response.body))
+            response.body.forEach((comment) => {
+                expect(comment.hasOwnProperty('comment_id')).toBe(true)
```
Comment (roisinoreilly) [inline __tests__/intergration.test.js:90]: try using .toMatchObject or .objectContaining here

File: __tests__/intergration.test.js [hunk 2]
```diff
@@ -66,5 +66,38 @@ describe('/api/articles/:article_id', () => {
         });
     });
 })
-
+describe('/api/articles/:article_id/comments', () => {
+    it('should return comments of article based on id', () => {
+        return request(app).get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+            console.log(response.body);
```
Comment (roisinoreilly) [inline __tests__/intergration.test.js]: don't forget to remove stray logs

File: __tests__/intergration.test.js [hunk 3]
```diff
@@ -66,5 +66,38 @@ describe('/api/articles/:article_id', () => {
...
+                expect(comment.hasOwnProperty('comment_id')).toBe(true)
+                expect(comment.hasOwnProperty('body')).toBe(true)
+                expect(comment.hasOwnProperty('article_id')).toBe(true)
+                expect(comment.hasOwnProperty('author')).toBe(true)
+                expect(comment.hasOwnProperty('votes')).toBe(true)
+                expect(comment.hasOwnProperty('created_at')).toBe(true)
+            })
+        })
+    });
+    // it('GET:404 sends an appropriate status and error message when given a valid but non-existent id', () => {
```
Comment (roisinoreilly) [inline __tests__/intergration.test.js]: once you've merged the previous branch can you uncomment these and send us another PR

**Review by roisinoreilly — CHANGES_REQUESTED** [review]
nice job so far, can you uncomment the tests you were waiting on the previous PR for and send us another one so husky runs them. I'd also like you to add in a 200 test for an article that exists but has no comments, I think article id 2 has none. Once you've done that we'll get it approved
